AppLovin MAX Demo App - Swift/AppLovin MAX Demo App - Swift/Base Classes/ALHomeViewController.swift

```diff
@@ -101,13 +101,13 @@ class ALHomeViewController: UITableViewController
          * Toggling the sdk mute setting will affect whether your video ads begin in a muted state or not.
          */
         let sdk = ALSdk.shared()
-        sdk?.settings.muted = !(sdk?.settings.muted)!
+        sdk?.settings.isMuted = !(sdk?.settings.isMuted)!
```

Comment:
muted 's getter is isMuted. this will cause build error

Reply:
Thx for fixing - can you do this in another PR since we won't merge the new impression callback changes yet.

Reply:
Will do!